core/src/main/java/com/google/zxing/oned/Code93Writer.java

```diff
@@ -40,74 +40,49 @@ public BitMatrix encode(String contents,
 
   @Override
   public boolean[] encode(String contents) {
+    contents = convertToExtended(contents);
```

Comment:
We probably need a comment in the javadoc here explaining that the contents should _not_ already be encoded for extended characters. Otherwise this would double-encode them.